src/Object/Api/Mastodon/Instance.php

```diff
@@ -84,14 +84,14 @@ public function __construct(IManageConfigValues $config, BaseURL $baseUrl, Datab
 	{
 		$register_policy = intval($config->get('config', 'register_policy'));
 
-		$this->uri               = $baseUrl;
+		$this->uri               = $baseUrl->getHost();
 		$this->title             = $config->get('config', 'sitename');
 		$this->short_description = $this->description = $config->get('config', 'info');
 		$this->email             = implode(',', User::getAdminEmailList());
 		$this->version           = '2.8.0 (compatible; Friendica ' . App::VERSION . ')';
 		$this->urls              = null; // Not supported
 		$this->stats             = new Stats($config, $database);
-		$this->thumbnail         = $baseUrl . 'images/friendica-banner.jpg';
+		$this->thumbnail         = $baseUrl->__toString() . '/images/friendica-banner.jpg';
```

Comment:
I thought adding a text to a class automatically executes the `__toString()` method. But if not, I`d prefer it like:

```suggestion
		$this->thumbnail         = (string)$baseUrl . '/images/friendica-banner.jpg';
```
But as u like :-)